stylus/utilities/colors.styl

```diff
@@ -0,0 +1,23 @@
+utility_colors = 'black',
+   'charcoalGrey',
+   'coolGrey',
+   'silver',
+   'slateGrey',
+   'danger',
+   'lightishPurple',
+   'dodgerBlue',
+   'error',
```

Comment:
Is it normal to have both `danger` and `error`?

Reply:
It was "known" that we had the two colors, I removed "danger", thanks.